libs/application/templates/aosh/transfer-of-machine-ownership/src/lib/TransferOfMachineOwnershipTemplate.ts

```diff
@@ -0,0 +1,356 @@
+import {
+  ApplicationConfigurations,
+  ApplicationTemplate,
+  ApplicationTypes,
+  ApplicationContext,
+  ApplicationRole,
+  ApplicationStateSchema,
+  Application,
+  DefaultEvents,
+  defineTemplateApi,
+  PendingAction,
+  InstitutionNationalIds,
+} from '@island.is/application/types'
+import {
+  EphemeralStateLifeCycle,
+  coreHistoryMessages,
+  corePendingActionMessages,
+  getValueViaPath,
+  pruneAfterDays,
+} from '@island.is/application/core'
+import { Events, States, Roles } from './constants'
+import { ApiActions } from '../shared'
+import { AuthDelegationType } from '@island.is/shared/types'
+import { MachineAnswersSchema } from './dataSchema'
+import { application as applicationMessage } from './messages'
+import { assign } from 'xstate'
+import set from 'lodash/set'
+import {
+  IdentityApi,
+  UserProfileApi,
+  VinnueftirlitidPaymentCatalogApi,
+  MachinesApi,
+} from '../dataProviders'
+import { getChargeItemCodes, hasReviewerApproved } from '../utils'
+import { buildPaymentState } from '@island.is/application/utils'
+import { ApiScope } from '@island.is/auth/scopes'
+
+const pruneInDaysAtMidnight = (application: Application, days: number) => {
+  const date = new Date(application.created)
+  date.setDate(date.getDate() + days)
+  const pruneDate = new Date(date.toUTCString())
+  pruneDate.setHours(23, 59, 59)
+  return pruneDate
+}
+
+const determineMessageFromApplicationAnswers = (application: Application) => {
+  const regNumber = getValueViaPath(
+    application.answers,
+    'machine.regNumber',
+    undefined,
+  ) as string | undefined
+  return {
+    name: applicationMessage.name,
+    value: regNumber ? `- ${regNumber}` : '',
+  }
+}
+
+const reviewStatePendingAction = (
+  application: Application,
+  role: string,
+  nationalId: string,
+): PendingAction => {
+  if (nationalId && !hasReviewerApproved(nationalId, application.answers)) {
+    return {
+      title: corePendingActionMessages.waitingForReviewTitle,
+      content: corePendingActionMessages.youNeedToReviewDescription,
+      displayStatus: 'warning',
+    }
+  } else {
+    return {
+      title: corePendingActionMessages.waitingForReviewTitle,
+      content: corePendingActionMessages.waitingForReviewDescription,
+      displayStatus: 'info',
+    }
+  }
+}
+const template: ApplicationTemplate<
+  ApplicationContext,
+  ApplicationStateSchema<Events>,
+  Events
+> = {
+  type: ApplicationTypes.TRANSFER_OF_MACHINE_OWNERSHIP,
+  name: determineMessageFromApplicationAnswers,
+  institution: applicationMessage.institutionName,
+  //featureFlag: Features.transferOfMachineOwnership,
```

Comment:
Is this not going out under a feature flag?

Reply:
Yes, nice stop will add it back in latest changes